Week3/script.js

```diff
@@ -250,24 +257,21 @@ const o1 = o2;
 
 */
 
-console.log(o1 == o2);  // true
-console.log(o2 == o3);  // false
-console.log(o3 == o1);  // false
-console.log(o2 == o1);  // false
+console.log(o1 == o2); // true
+console.log(o2 == o3); // false
+console.log(o3 == o1); // false
+console.log(o2 == o1); // false
 
 // when we change o2 whit o3, giving syntax error
 
 // Does the order that you assign (o3 = o2 or o2 = o3) matter?
 
-// sure. when you use o2 = o3, o2 declares twice. giving syntax error.  
-
-
-
+// sure. when you use o2 = o3, o2 declares twice. giving syntax error.
```

Comment:
Ah yeah true, but in the assignment the code is as follows:

```
let o1 = { foo: "bar" };
let o2 = { foo: "bar" };
let o3 = o2;
```

And you did `const` then it's true that an error will occur :) but if you use `let`, will the order that you assign o3 = o2 or o2 = o3 matter?

And if you change o2, will it change o3? 
And if you change o3, will it change o2?